packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -1071,6 +1085,16 @@ interface InvertContext {
 
 interface RebaseTable extends CrossFieldTable<FieldChange> {
 	rebasedFieldToContext: Map<FieldChange, FieldChangeContext>;
+	/**
+	 * The key can be one of:
+	 * - The node changeset that is in the input changeset being rebased
+	 * - The node changeset that is in the base changeset (used when there is no corresponding node changeset in the input changeset)
```

Comment:
"The key can be one of" makes it sound like either key could be used arbitrarily, but it's very important that the key is always from the input changeset if the input has a changeset for the node.